pvlib/iam.py

```diff
@@ -274,6 +274,111 @@ def martin_ruiz(aoi, a_r=0.16):
     return iam
 
 
+def martin_ruiz_diffuse(slope, a_r=0.16, c1=0.4244, c2=None):
+    '''
+    Determine the incidence angle modifiers (iam) for diffuse sky and
+    ground-reflected irradiance using the Martin and Ruiz incident angle model.
+
+    Parameters
+    ----------
+    slope : numeric, degrees
+        The slope or tilt angle of the module, where 0 is horizontal.
+
+    a_r : numeric
+        The angular losses coefficient described in equation 3 of [1].
+        This is an empirical dimensionless parameter. Values of a_r are
+        generally on the order of 0.08 to 0.25 for flat-plate PV modules.
+        a_r must be a positive numeric scalar.
+
+    c1, c2 : numeric scalar
+        Two fitting parameters for the expressions that approximate the
+        integral of diffuse irradiance coming from different directions.
+        c1 is contant at 4 / 3 / pi (0.4244) and c2 varies with a_r.  The
+        calculation of c2 from a_r is according to [3].
+
+    Returns
+    -------
+    iam_sky : numeric
+        The incident angle modifier(s) for sky diffuse
+
+    iam_ground : numeric
+        The incident angle modifier(s) for ground-reflected diffuse
+
+    Notes
+    -----
+    Sky and ground modifiers are complementary: iam_sky for slope = 30 is
+    equal to iam_ground for slope = 180 - 30.  For vertical surfaces,
+    slope = 90, the two factors are equal.
+
+    References
+    ----------
+    [1] N. Martin and J. M. Ruiz, "Calculation of the PV modules angular
```

Comment:
I think the primary reference should be

[1] Martín, N., Ruiz, J. M. 2005. Annual angular reflection losses in PV modules. Progress in Photovoltaics: Research and Applications, 13(1), 75–84.  The corrigendum [2] doesn't apply.

Reply:
I disagree.  The typos may not be there, but neither are all the explanations.

Reply:
You are correct, the FA and FD equations are in the 2001 paper.  Disregard this comment.